core/src/coreLinkDef.h

```diff
@@ -16,6 +16,7 @@
 #pragma link C++ class genfit::PruneFlags+;
 #pragma link C++ class genfit::TrackCand+;
 #pragma link C++ class genfit::TrackCandHit+;
+#pragma link C++ class genfit::SharedPlanePtrCreator+;
```

Comment:
Do we need a "+" here? Was this also tested with "-"?

Reply:
I think we did not test it for this solution for getting the SharedPlanePtr. However, I guess the streamer is not necessary here. I can update and test this.